plugins/generated.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs, lib, config, ... }:
```

Comment:
Is this just for small plugins that don't define any options?

Reply:
This is used to quickly add new plugins as a module, if there is currently no time to write the complete plugin.
This way there people can already add this to their config, and update if the module is completed.

As you said, some plugins that do not require any config, could be left in this list